uniffi_aries_vcx/core/src/vcx.udl

```diff
@@ -37,32 +37,6 @@ dictionary PairwiseInfo {
     string pw_vk;
 };
 
-dictionary AriesDidDoc {
```

Comment:
just checking, I guess this is what you were discussing with George on Discord, right?

Reply:
lgtm, matches my interpretation of the conversation. Moving back to using `string` for aries_did_doc due to uniffi typing issues that are outside the scope of the PR to fix.

Reply:
It is, yes.